source/_integrations/refoss.markdown

```diff
@@ -26,6 +26,10 @@ Integrate Refoss devices into Home Assistant.
 - The integration will occupy port: 9989.    
 {% include integrations/config_flow.md %}
 
+### Supported device model
+
+- r10
```

Comment:
```suggestion
- Refoss Smart Wi-Fi Switch, R10
```
would it make sense here to use the full product name? That's how I found it on their website, but please correct me if I got it wrong.

Reply:
The device model is r10，   Refoss Smart Wi-Fi Switch   is just  a classification.

Reply:
You're right,    users can better understand